### DISCUSSION TITLE
Default style for components

### DISCUSSION THREAD

**Discussion by manufunk** [tamagui/tamagui#886] [General]
### Current Behavior

Is there any way to change the default style of a component without the need to come up with a custom component `const myButton = styled(Button,...)`?
For example I want to set the cursorColor somewhere in the theme globally. It is easy to add just "cursorColor" to the component but it would be nicer to set this initially via the theme. So far I did not find a good description about whats supported in the "createTheme" method. Maybe I'm just missing the right config name? Also looking for a way to define variants on the main component as I'm not a big fan on wrapping default component in custom components.

### Tamagui Version

```markdown
1.9.22
```

### Reproduction

```markdown
Just a question
```

**Comment by natew** [comment]
Converted to discussion!

**Comment by kevinchristianto** [comment]
Is it possible now?
Having the option to change the default components style without having to define a new component is really nice to have, looking forward to that feature.

**Reply by vanenshi** [reply]
+ same here

**Comment by bernard-ng** [comment]
any news on this one ?
